schema/codelists/itemClassificationScheme.csv

```diff
@@ -7,3 +7,6 @@ CPC,Central Product Classification,"The Central Product Classification (CPC) is
 OKDP,Russian National Classification of Products by Economic Activity (1994),"Part of the unified system of classification and coding of technical, economic and social information of the Russian Federation, in use for Russian Procurement information until January 1, 2016. Superseded by OKPD.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=143095;fld=134;from=14052-7;rnd=0.33374630799517035
 OKPD,Russian National Classification of Products by Economic Activity (2014),"Part of the national standardization system of the Russian Federation and based on harmonization with the Statistical Classification of Products by Activity in the European Economic Community, 2008. In use from January 1 2014.",http://base.consultant.ru/cons/cgi/online.cgi?req=doc;base=LAW;n=163703;fld=134;from=170864-15;rnd=0.39699640963226557
 CUCOP,Mexican Single Classifier of Public Procurement,"The Single Classifier of Public Procurement (Clasificador Único de las Contrataciones Públicas) is a national scheme in Mexico for the classification of goods, works, services and leases.",https://sites.google.com/site/cnetcucop/
+NAICS,North American Industry Classification System, "The North American Industry Classification System (NAICS) is the standard used by Federal statistical agencies in classifying business establishments for the purpose of collecting, analyzing, and publishing statistical data related to the U.S. business economy.",https://www.census.gov/eos/www/naics/downloadables/downloadables.html
+PSC,Product and Service Codes,"Also referred to as federal supply codes, product service codes are used by the United States government to describe the products, services, and research and development purchased by the government.",https://www.fpds.gov/downloads/service_product_codes.pdf
+HS,Harmonized Commodity Description and Coding System,"The Harmonized System is an international nomenclature for the classification of products. It allows participating countries to classify traded goods on a common basis for customs purposes. At the international level, the Harmonized System (HS) for classifying goods is a six-digit code system. Other names used in the past for this were Customs Cooperation Council Nomenclature (CCCN) and Brussels Tariff Nomenclature.",https://unstats.un.org/unsd/tradekb/Knowledgebase/50018/Harmonized-Commodity-Description-and-Coding-Systems-HS
```

Comment:
It seems like the relationship to CCCN and BTF is more complicated (there are many pages [here](http://harmonizedsystem.wcoomdpublications.org/pdfs/WCOOMD_MSH_EN.pdf) about it), so let's cut the last sentence.

Reply:
done